lib/cc/cli/config_generator.rb

```diff
@@ -17,6 +17,10 @@ def initialize(filesystem, engine_registry)
         @engine_registry = engine_registry
       end
 
+      def can_generate?
+        true
```

Comment:
For the `UpgradeConfigGenerator`: the stub is here for API consistency between the two.